cms/models/placeholdermodel.py

```diff
@@ -111,52 +111,35 @@ def _get_url(self, key, pk=None):
             model_name = model.__name__.lower()
             return admin_reverse('%s_%s_%s' % (app_label, model_name, key), args=args)
 
-    def _get_permission(self, request, key):
+    def _has_permission(self, user, key):
```

Comment:
This looks like a new method, yet if declares params that will never be used. Intentional?